casper/src/main/scala/coop/rchain/casper/util/comm/DeployRuntime.scala

```diff
@@ -13,6 +13,9 @@ object DeployRuntime {
   //force Casper to propose a block
   def forcePropose[F[_]: DeployService]: F[Unit] = DeployService[F].propose()
 
+  def showBlock[F[_]: Functor: DeployService](hash: String): F[Unit] =
+    DeployService[F].showBlock(BlockQuery(hash)).map(println(_))
```

Comment:
Aren't you using sth like `Log[F].log(…)` ?

Reply:
The `Log` implicit instance we have today is used server-side, but this is a client-side gRPC interface, so I want to return the result directly to the user.

Reply:
Ah sorry, didn't read carefully. :+1: